lib/qs/payload_handler.rb

```diff
@@ -88,9 +88,9 @@ def log_complete(redis_item)
         'time'    => redis_item.time_taken,
         'handler' => redis_item.handler_class
       }
-      if (job = redis_item.job)
-        summary_line_args['job']    = job.name
-        summary_line_args['params'] = job.params
+      if (message = redis_item.message)
+        summary_line_args['name']   = message.route_id
```

Comment:
@jcredding yeah, I agree on exposing route id in the logs.  We discussed handling this by doing event/job specific logging in a later effort.